addons/analytic/models/analytic_plan.py

```diff
@@ -22,7 +22,7 @@ def _default_color(self):
         string="Parent",
         ondelete='cascade',
         check_company=True,
-        domain="[('id', '!=', id)]",
+        domain="[('id', '!=', id), '!', ('id', 'child_of', id)]",
```

Comment:
a record is child_of and parent_of itself, no need to keep the original domain [('id', '!=', id)], I think :thinking:

Reply:
Indeed, I've just checked.